### ISSUE TITLE
VIB selection question

### ISSUE THREAD

**Issue by aCoalBall** [invoker-LL/WSI-finetuning#10]
Hi, I am confused about the vib patch-selection process in the code. In the paper it mentioned that the patches are selected based on the bernouli probability, but I found that in the code it is actually selected based on the attention scores from CLAM. I tested the code and the performance is not good. Could you help me to solve this? Thanks.

`logits, Y_prob, Y_hat, A, results_dict = model_wsi(features, testing=True)`
`topk_indexes = torch.topk(A.squeeze(), min(args.topk_num,A.shape[1]), sorted=True)[1].cpu()`

**Comment by invoker-LL** [comment]
Sorry, this is a bug. I should have return the vib probability (Attn_score in model vib_clam_wsi.py) instead as masked A. I will commit this modification soon.
